test/convert_test.go

```diff
@@ -86,4 +84,32 @@ func TestConvert(t *testing.T) {
 			)
 		})
 	}
+
+	validationTests := []struct {
+		desc            string
+		policyPath      string
+		wantError       bool
+		wantOutputRegex string
+	}{
+		{"always_violate", "sample_policies/always_violate", true, "Constraint always_violates_all on resource"},
```

Comment:
nit: can we use the `field: value` form for the struct literal? it uses up more vertical space, but I find it easier to follow along